internal/trie/node/decode.go

```diff
@@ -39,21 +38,22 @@ func Decode(reader io.Reader) (n *Node, err error) {
 	}
 
 	switch variant {
-	case leafVariant.bits:
-		n, err = decodeLeaf(reader, partialKeyLength)
+	case emptyVariant:
+		return nil, nil //nolint:nilnil
+	case leafVariant, leafWithHashedValueVariant:
```

Comment:
For certain parachain subsystems, we need to be able to get the leaf updates. I don't know anything about it yet, but just wondering if we have some sort of broadcast channel for it already

Reply:
That's a good point! good to know, do we have an issue for this? I think we could add it in a new PR

Reply:
No, not yet, we're still exploring. I guess I'll reach out to you if we need help, thank you!